utils/questGen.py

```diff
@@ -77,3 +72,104 @@ def pokemonname(id):
     with open('pokemon.json') as f:
         mondata = json.load(f)
     return mondata[str(int(id))]["name"]
+
+def questtask(typeid, condition, target):
+    with open('utils/quest/pokemonTypes.json') as f:
+        pokemonTypes = json.load(f)
+    with open('utils/quest/items.json') as f:
+        items = json.load(f)
+    throwTypes = {"10":"Nice", "11":"Great", "12":"Excellent", "13":"Curveball"}
+    arr = {}
+    arr['0'] = target
+    text = questtype(typeid)
```

Comment:
You load the text, but replace it in all instances???

Look at type id 5 or 6 for example. Those elifs are not needed?

Reply:
I load the text, so if my code wont edit it the default text will be returned.